.env

```diff
@@ -1 +1,3 @@
-REACT_APP_API_BASE_PATH=http://localhost:8000/api/v1
\ No newline at end of file
+REACT_APP_API_BASE_PATH=http://localhost:8000/api/v1
+REACT_APP_SOCKET_BASE_PATH=http://localhost:8000
+REACT_APP_SOCKET_BASE_PATH= https://musicstore.onrender.com
```

Comment:
We only need to have our dev variables here in the \`.env\` file.

By including \`REACT\_APP\_SOCKET\_BASE\_PATH\` twice here, the second value will overwrite the first one. So when you run your app locally, it will be trying to connect to the production server:

![](https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/4a5238fb-b355-4b8a-9b75-67d8d3943f53)

For production, just like with our backend server, it doesn't use the `.env` file. Rather, it will be getting the environment variables from those injected by Render.com when the app is started. We provide those variables in the Render.com settings pane for the frontend app: https://dashboard.render.com/static/srv-cjf0j06e546c73d6lq10/env

Reply:
Removed. I see now, we have on render this, thank you for explanation.